syft/__init__.py

```diff
@@ -131,3 +131,7 @@ def pool():
     from syft.generic.id_provider import IdProvider
 
     ID_PROVIDER = IdProvider()
+
+
+def manual_seed(seed=0):
```

Comment:
@karlhigley is this a good place for this? I'm stumped as to where this has to go :thinking:

Reply:
It looks like it's only used once, so maybe it could be in-lined in the torch hook?